scalafmt-tests/src/test/resources/default/Try.stat

```diff
@@ -43,7 +43,9 @@ object Foo {
 >>>
 object Foo {
   try proc.waitFor()
-  catch { case e: Throwable => run() }
```

Comment:
I'm not sure this diff here would be popular, it's quite common to define a catch in one line. Is there a way to preserve the old formatting?

Reply:
i will try again. earlier, couldn't get it to work.

Reply:
done